client/core/status.go

```diff
@@ -235,8 +239,13 @@ func resolveMissedTakerAudit(dc *dexConnection, trade *trackedTrade, match *matc
 		return
 	}
 
+	// len(srvData.TxData) == 0 {
+	// 	err = fmt.Errorf("Server is reporting a match in TakerSwapCast with no tx data. %s", logID)
+	// 	return
+	// }
+
```

Comment:
Why is this one commented out?

I think this was commented somewhere else, but I guess the client will need to check that the server's version is compatible with itself. An older server would not include the `TxData`.

Reply:
This PR is just a part of a few steps to this work (next is presumably using the TxData with bcast), but it seems to me that backward compatibility regarding this TxData field is straightforward for both client and server.

- If the TxData is omitted/empty, the client should take the legacy approach of searching for the txn
- If the TxData is present, take the new approach of broadcasting it

I don't believe any versioning is required to handle this elegantly and without any hacks.

As such, I don't think this commented code will ever be uncommented given the legacy handling I'm suggesting if it is len 0.